tests/unit/backend/utils/field-validation/email-validation.spec.js

```diff
@@ -44,4 +44,99 @@ describe('Email field validation', () => {
     const testFunc = () => validateField('formId', formField, response)
     expect(testFunc).not.toThrow()
   })
+
```

Comment:
Please help to rewrite this in Typescript using `jest`! As spoken on slack, we can also mock `formsgSdk` to return true; or even `BaseFieldValidator._isSignatureValid` to return true for your failing tests.

Reply:
I will work on this. Thank you!